Toolbox/final/finalScript.sh

```diff
@@ -19,28 +19,27 @@ if [ -z "$VOLUME" ]; then
     fi
 fi
 
-# Make it writable
-on -f mmx /bin/mount -uw /mnt/app
+# Make SD writable for logging
+on -f mmx /bin/mount -uw $VOLUME
 
-# Copy scripts
-on -f mmx /bin/mkdir -p /net/mmx/mnt/app/eso/hmi/engdefs/scripts/mqb
-on -f mmx /bin/cp -r $VOLUME/Toolbox/scripts/*.sh /net/mmx/mnt/app/eso/hmi/engdefs/scripts/mqb
-on -f mmx /bin/chmod a+rwx /net/mmx/mnt/app/eso/hmi/engdefs/scripts/mqb
+LOGDIR=$VOLUME/Log
+LOGFILE=$LOGDIR/install_final.txt
 
-# Copy additional GreenMenus
-on -f mmx /bin/cp -r $VOLUME/Toolbox/GEM/*.esd /net/mmx/mnt/app/eso/hmi/engdefs
+mkdir -p $LOGDIR
 
-# Make readonly again
-on -f mmx /bin/mount -ur /mnt/app
+/bin/ksh $VOLUME/Toolbox/final/install_scripts.sh $VOLUME > $LOGFILE 2>&1
```

Comment:
Isn`t the `$VOLUME` in `$VOLUME > $LOGFILE 2>&1` too much?
In line 33 we have the same logic and not unsing a second `$VOLUME`

Reply:
The $VOLUME here is an argument being sent to install_scripts.sh, it's not related to the log